client/dom/sandbox.ts

```diff
@@ -74,7 +74,7 @@ export function newSandboxDiv(sandbox_holder: Selection<HTMLDivElement, any, any
 		.append('div')
 		.attr('class', 'sjpp-output-sandbox-header')
 		.style('width', opts.style?.width || '95vw')
-		.style('border', '1px solid grey')
+		.style('border', '1px solid #ccc')
```

Comment:
sandbox border color are defined at two places. will be nice if defined only in one